src/table/components/TableWrapper.tsx

```diff
@@ -69,8 +70,8 @@ export default function TableWrapper(props) {
     });
   };
 
-  useFocusListener(tableWrapperRef, shouldRefocus, keyboard);
-  useScrollListener(tableContainerRef, direction);
+  useFocusListener(tableWrapperRef as React.MutableRefObject<HTMLDivElement>, shouldRefocus, keyboard);
+  useScrollListener(tableContainerRef as React.MutableRefObject<HTMLDivElement>, direction);
```

Comment:
There are sanity checks in both hooks to take the possible undefined into account. Both refs could be undefined the first times it runs I think, so the casting here is a bit risky

Reply:
Fixed